src/app/views/query/query.component.js

```diff
@@ -32,25 +23,37 @@ class QueryController {
       'event.decimalLongitude',
     );
 
-    if (this.currentProject) {
-      this.handleProjectChange(this.currentProject);
-    }
-
     this.showSidebar = true;
     this.showMap = true;
     this.loading = false;
     this.sidebarToggleToolTip = 'hide sidebar';
   }
 
-  handleNewResults(results) {
+  handleNewResults(results, isAdvancedSearch) {
+    // when switching between simple and advanced searches
+    if (!results) {
+      this.revert = true; // clears table data
```

Comment:
I don't understand the use of revert. It looks like it is only used to display the query table `<div ng-show="$ctrl.results;!$ctrl.revert">`. Why not just use the results like was previously done and always call `this.results = results;` in this function?